engines/cegar_values.cpp

```diff
@@ -269,6 +288,11 @@ void CegarValues<Prover_T>::cegar_abstract()
   super::bad_ = va.visit(super::bad_);
   cegval_bad_ = to_cegval_solver_.transfer_term(super::bad_, BOOL);
 
+  // add polarity axioms from abstraction
+  for (const auto & ax : va.polarity_axioms()) {
+    prover_ts_.add_constraint(ax);
```

Comment:
is it possible to add this to trans only?

Reply:
Good idea. Added here 266b7d06c797951eba0cf3706ffc6f8fb430b57e. Does that look all right?